spec/factories/users.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    password { 'password' }
+
+    trait :email do
+      email { Faker::Internet.email }
+    end
+    trait :phone do
+      phone { Faker::Base.numerify('###-###-####') }
+    end
+    trait :avatar do
+      avatar { Rack::Test::UploadedFile.new('/home/developer/avatar.svg') }
+    end
+    trait :address_first_last_names do
+      address { Faker::Address.full_address }
+      first_name { Faker::Name.first_name }
+      last_name { Faker::Name.last_name }
+    end
+    trait :all_fields do
+      email
+      phone
+      avatar
+      address_first_last_names
+    end
+  end
+end
```

Comment:
you need to create a factory, which will build full object.
So, default factory `:user` should contain all required for user fields
traits needed, when you want to redefine something, for example:

```ruby
factory :user do
  name { 'name' }
  active { true }

  trait :inactive do
   # this trait inherits all fields from the parent one
   active { false }
  end
end
```